common/app/model/Formats.scala

```diff
@@ -75,7 +75,6 @@ object MetaDataFormat {
     contentType: String,
     hasHeader: Boolean,
     schemaType: Option[String],
-    cacheSeconds: Int,
```

Comment:
not sure if it's worth storing the cache time in the pressed json, unless I missed something?